### PR TITLE
Register warden nikku876

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden nikku876

@Kali-Decoder

### CODE REVIEW

File: _data/handles/nikku876.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "handle": "nikku876",
```
Comment (Kali-Decoder) [inline _data/handles/nikku876.json:2]: handle="nikku.jr.dev#1321"

**Comment by SamusElderg** [comment]
Can not find a matching user in Discord, will leave the application open for now though

**Comment by Kali-Decoder** [comment]
Actually i have to change username
it should be nikku.jr.dev

**Comment by SamusElderg** [comment]
> Actually i have to change username it should be nikku.jr.dev

All you need to do is change your nickname in the Code4rena Discord server to: `nikku876`. You do not need to change your global Discord handle/username (only your local nickname in the C4 Discord server)

**Comment by SamusElderg** [comment]
Closing this PR for now, we can re-open once nickname is changed to match

**Pull request closed without merging**